docs/pywbemcli/commands.rst

```diff
@@ -1556,6 +1560,57 @@ Example:
 
     $ pywbemcli --name myserver server remove-mof mymodel.mof
 
+See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
+command.
+
+.. _`Server schema command`:
+
+``server schema`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``server schema`` command returns information on the :ref:`DMTF Schema`
+defined for classes in the namespace()s) of the WBEM server of the
+:term:`current connection`.  It provides information on the maximum value class
+version qualifier (which may represent the DMTF ) in the namespace, whether the
+namespace has experimental classes, the number of classes in the schema, and
+the set of schema prefixes.
+
+pywbemcli cannot determine the actual DMTF schema version of classes that was
+installed in each namespace because the version of the DMTF schema installed is
+not recorded in the installed model so the best that can be done is to find
+the highest value of the Version qualifier that represents the DMTF Schema
+version where the last change was made to that each class.
```

Comment:
This sounds quite negative. I would phrase it more neutrally:

"Pywbemcli determines the version of a schema by the highest value of the Version qualifier on any of its classes."

Reply:
DONE.